src/validation/schemas/modal/modal-open-request.v2.schema.ts

```diff
@@ -0,0 +1,80 @@
+export const modalOpenRequest_v2_schema = {
+  type: 'object',
+  properties: {
+    url: {
+      type: 'string',
+    },
+    modalSettings: {
+      type: 'object',
+      properties: {
+        title: {
+          type: 'string',
+        },
+        showTitleHeader: {
+          type: 'boolean',
+        },
+
+        hasBackdrop: {
+          type: 'boolean',
+        },
+        backdropClickCloseable: {
+          type: 'boolean',
+        },
+        escKeyCloseable: {
+          type: 'boolean',
+        },
+        focusTrapped: {
+          type: 'boolean',
+        },
+        fullScreen: {
+          type: 'boolean',
+        },
+        mobile: {
+          type: 'boolean',
+        },
+        mobileOuterSpacing: {
+          type: 'boolean',
+        },
+        draggable: {
+          type: 'boolean',
+        },
+        resizable: {
+          type: 'boolean',
+        },
+        verticalPadding: {
```

Comment:
`verticalPadding` and `responsivePadding` do not do anything right now for our use case.
I have kept them for consistency with the FU API, yet I have chosen not to document them, as they do not do anything.
Feel free to state if you believe I should provide otherwise.